docs/tutorials/tabular_prediction/tabular-gpu.md

```diff
@@ -41,19 +41,47 @@ hyperparameters
 
 ### Text models
 
-Text model preset to use can be set via:
+Text model's configuration can be set via:
 
 ```{.python}
-hyperparameters['AG_TEXT_NN'] = ['<preset>']
+hyperparameters['AG_TEXT_NN'] = {
+    "model": "/path/to/model/yaml/file",
+    "data": "/path/to/data/yaml/file",
+    "optimization": "/path/to/optimization/yaml/file",
+    "environment": "/path/to/optimization/yaml/file",
+}
```

Comment:
I noticed that later you mentioned to override certain keys with 'model.hf_text.checkpoint_name': 'google/electra-small-discriminator'. What’s the difference between these two ways of setting the hyperparameters?

Reply:
This is actually an API breaking change. User's old code will no longer work with newer version of AG. We should still support preset along with the custom configuration.

Reply:
Do you think adding a AutoMM tutorial is a solution? The previous three presets were only temporary since they provide users very limited flexibility.

Reply:
Will a tutorial in the next PR.

Reply:
The preset is back now.